src/main/java/com/sprint/mission/discodeit/entity/UserStatus.java

```diff
@@ -0,0 +1,41 @@
+package com.sprint.mission.discodeit.entity;
+
+import java.io.Serializable;
+import java.time.Duration;
+import java.time.Instant;
+import java.util.UUID;
+import lombok.Getter;
+
+@Getter
+public class UserStatus extends BaseEntity implements Serializable {
+    private static final long serialVersionUID = 1L;
+    private final UUID userId;
+
+    private Instant lastActiveAt;
+
+    public UserStatus(UUID userId, Instant lastActiveAt) {
+        super();
+        this.userId = userId;
+        this.lastActiveAt = lastActiveAt;
+    }
+
+    public void update(Instant lastActiveAt) {
+        boolean anyValueUpdated = false;
+        if (lastActiveAt != null && !lastActiveAt.equals(this.lastActiveAt)) {
+            this.lastActiveAt = lastActiveAt;
+            anyValueUpdated = true;
+        }
+
+        if (anyValueUpdated) {
+            updateUpdatedAt();
+        }
+    }
+
+    public boolean isActive() {
+        Instant now = Instant.now();
+        Instant lastActive = this.lastActiveAt;
+        Duration duration = Duration.between(now, lastActive);
+
+        return duration.getSeconds() < 300; // 5분 이내면 true;
```

Comment:
magic number보다는 별도 static 빼서 처리하는게 더 관리하기 좋습니다. 
5분이라는 것은 비즈니스 로직이기 때문입니다.
```suggestion
    private static final Duration MAX_ACTIVE_MINUTE = Duration.ofMinutes(5); // 이렇게 상수로 빼고
    
       
          return duration.compareTo(MAX_ACTIVE_MINUTE) < 0; // 5분 이내면 true;
       
```
MAX_ACTIVE_MINUTE 는 좀 생각해보시고.. 

이 글을 참고해보세요!
https://hoonmaro.tistory.com/44

Reply:
300 자체를 상수형 변수로 빼는 방법도 있습니다.  MAX_ACTIVE_SECONDS 처럼 단위까지 포함해서요!